_posts/0000-01-02-kishimaru13.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
# This is an <h1> tag
## This is an <h2> tag
Hello
###### This is an <h6> tag